src/Gateway.php

```diff
@@ -1332,8 +1332,8 @@ public function process_refund( $order_id, $amount = null, $reason = '' )
 
             $callbacks = $this->create_redirect_url( $order );
 
-            $url->setSuccess( $callbacks->getSuccess() . '?refund_callback=success&refund_unique_id=' . $refund_unique_id . '&order_id=' . $order_id )
-                ->setCancel( $callbacks->getSuccess() . '?refund_callback=cancel&refund_unique_id=' . $refund_unique_id . '&order_id=' . $order_id );
+            $url->setSuccess( $callbacks->getSuccess() . '&refund_callback=success&refund_unique_id=' . $refund_unique_id . '&order_id=' . $order_id )
+               ->setCancel( $callbacks->getSuccess() . '&refund_callback=cancel&refund_unique_id=' . $refund_unique_id . '&order_id=' . $order_id );
```

Comment:
Thease changes will break url if there's no paramaeters present.
This need to check if there's query parameters or not in url, and then add either ? or & before query additional parameters.

Reply:
@VuorelaJ-P Is there any built-in WP functionality for adding parameters to an url?